tests/StorageApiTestCase.php

```diff
@@ -704,4 +849,26 @@ private function createOrReuseDevBranch(self $that, $useExistingBranch = false)
 
         return $branch;
     }
+
+    /**
+     * @param string $name
+     * @return string
+     */
+    public function generateUniqueNameForString($name)
```

Comment:
Proč to nefunguje standardně? Tj. aby to poskládalo název testu atp. Takhle je to unikátní jen v rámci projektu - tzn. když ten projekt přepneme do parallel, tak to nebude fungovat a budou si sahat pod ruce

Reply:
A vlastně proč je potřeba mít specální metodu a nestačí `\Keboola\Test\StorageApiTestCase::generateDescriptionForTestObject`?

Reply:
Akorát mi přišlo škoda tam nemít čitelné to "name" tak jsem ho vytahl ven z sha1 (jako u bucketů máme in.c-api-tests + unikátní hash)